tests/__main__.py

```diff
@@ -5,6 +5,9 @@
 import subprocess
 import sys
 import unittest
+from collections import namedtuple
```

Comment:
It's pretty conventional to keep imports in alphabetical order (by module name).

Reply:
Of course. Fixed!